components/angular-tomitribe-select/src/tomitribe-select.ts

```diff
@@ -103,22 +103,33 @@ module tomitribe_select {
         function link(scope, element, attrs, uiSelect) {
             scope.openOnFocusDelay = angular.isDefined(attrs.openOnFocusDelay) ? parseInt(attrs.openOnFocusDelay) : 0;
             scope.openOnFocusDelayOnce = angular.isDefined(attrs.openOnFocusDelayOnce) ? attrs.openOnFocusDelayOnce : false;
+            const multiple = angular.isDefined(attrs.multiple);
```

Comment:
you already have uiSelect.multiple

Reply:
Yep, i know about it. But it's always ```undefined```. Looks like a bug.

Reply:
![image](https://user-images.githubusercontent.com/6061000/30163726-b09185ce-93e2-11e7-9780-0a14bb022f52.png)